doc/distrib/Open Source Licenses/SIF OFL.txt

```diff
@@ -0,0 +1,18 @@
+Code: MIT License (https://opensource.org/licenses/MIT)
+In the Font Awesome Free download, the MIT license applies to all non-font and
+non-icon files.
+Copyright 2023 Fonticons, Inc.
```

Comment:
This is not the SIF OFL license - this text needs to be added to the license and about box, and this file should be the entire text of the SIF OFL license you need to go find that somewhere, the text is not included in the task.

Reply:
https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL#5667e9e4

Reply:
Oh Thanks!